recipes/bohra/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "bohra" %}
+{% set version = "1.0.5" %}
+{% set sha256 = "f14fecdbb993114efbe059aa5e7810fbf3e397d13c2cc41141fbc74ea3da814a" %}
+
+package:
+  name: '{{ name }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/b/{{name}}/{{name}}-{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - '{{ compiler("c") }}'
+    - '{{ compiler("cxx") }}'
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - biopython >=1.70
+    - pathlib
+    - numpy
+    - svgwrite
+    - psutil
+    - pytest
+    - sh
+    - packaging
+    - datetime
```

Comment:
Is this in the standard python 3.7 library?

Reply:
I can only see `datetime` in the standard library list: https://docs.python.org/3/library/